pvfactors/irradiance/models.py

```diff
@@ -105,48 +127,70 @@ def transform(self, pvarray, idx=0):
             Index of the irradiance values to apply to the PV array (in the
             whole timeseries values)
 
+        Returns
+        -------
+        irradiance_vec : numpy array
+            List of summed up non-reflective irradiance values for all surfaces
+            and sky
+        rho_vec : numpy array
+            List of reflectivity values for all surfaces and sky
+        invrho_vec : numpy array
+            List of inverse reflectivity for all surfaces and sky
+        total_perez_vec : numpy array
+            List of total perez transposed irradiance values for all surfaces
+            and sky
         """
 
         for seg in pvarray.ground.list_segments:
             seg._illum_collection.update_params(
```

Comment:
Should parameters of `seg` be public if they are meant to be written from outside?

Reply:
yes you're right it's ambiguous here and I should just call the ``illum_collection`` property directly